packages/transpiler/src/transformations/fix-import-directives.ts

```diff
@@ -0,0 +1,20 @@
+import { getImportDirectives, getSourceIndices } from '../solc/ast-utils';
+import { Artifact } from '../solc/artifact';
+import { Transformation } from '../transformation';
+
+export function fixImportDirectives(artifact: Artifact, artifacts: Artifact[], contracts: string[]): Transformation[] {
+  const imports = getImportDirectives(artifact.ast);
+  return imports.map(imp => {
+    const [start, len] = getSourceIndices(imp);
+    const isTranspiled = artifacts.some(
+      art => art.ast.id === imp.sourceUnit && contracts.some(contract => contract === art.contractName),
+    );
```

Comment:
I'm not sure that this is correct. Let's say I have a file `A.sol` like this:

```solidity
import './B.sol';
contract A is B { ... }
```

I'm assuming that it would be possible to call `transpile(['A'], [ artifactForA, artifactForB ])`. In that case `B` has to be transpiled because `A` inherits from `B`, but this definition of `isTranspiled` would result in `false` for `import './B.sol'`.

Reply:
Yep. It is a bit implicit. I've tried to signal that with the name `contractsWithInheritance` but clearly it didn't work. I am open to better name for `contractsWithInheritance`.